assets/sass/components/_Footer.scss

```diff
@@ -1,71 +1,109 @@
 /*
  * Footer component
  */
-.Footer {
+footer {
   width: 100vw;
-  max-width: 100%;
-  border-top: 2px solid $menuBorderColor;
-
-  a {
-    display: block;
-  }
+  background-color: #ddd;
 
   p {
     display: block;
     margin: auto;
-    padding: 0.2em 0;
+    padding: 0.3em 0;
   }
-}
 
-/*
- * Footer--fixed component
- * 
- * This component can be used as a footer, fixed to the bottom of a page.
- */
-.Footer--fixed {
-  @extend .Footer;
-  position: fixed;
-  // always at the bottom
-  bottom: 0;
-  left: 0;
+  a {
+    color: $primaryColor;
+    text-decoration: none;
+    &:hover {
+      text-decoration: underline;
+    }
+  }
 }
 
 .Footer-Content {
   margin: auto;
 
   display: flex;
-  justify-content: center;
+  justify-content: space-between;
   flex-flow: row wrap;
 
 
   div{
-    width: 22em;
-    padding: 1em;
+    width: 10em;
+    padding: 2em 0 3em;
 
     h3{
       margin: 0 auto;
+      padding-bottom: 0.5em;
     }
   }
 
 }
 
+$footerContentDim: 0.5;
+$footerBackgroundColor: #555;
+
 .Footer-Bottom {
-  border-top: 2px solid $menuBorderColor;
 
-  padding: 1em;
+  background: $footerBackgroundColor;
+  color: rgb(255 * $footerContentDim, 255 * $footerContentDim, 255 * $footerContentDim);
 
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  flex-wrap: wrap;
+  .Footer-Bottom-Content {
+    display: flex;
+    justify-content: space-between;
+    flex-flow: row wrap;
+    flex-basis: auto;
+    padding-top: 0;
+    padding-bottom: 0;
 
-  text-align: center;
+    $logoWidth: 5em;
+    $textWidth: 24em;
+    @media screen and (max-width: $logoWidth + 2*$textWidth + 2*$containerSidePadding) {
+      justify-content: center;
+      flex-flow: column;
+      align-items: center;
+    }
 
-  div {
-    padding: 0.5em 0.8em;
-  }
+    a, svg {
```

Comment:
Please use explicit classes respecting BEM. I now this was not respected in other Pull-Requests but I suspect our CSS at the moment with all this nesting occurring is not maintainable anymore. We need to strive for a better structure and cleanup everything.